x-pack/test/apm_api_integration/tests/transactions/transactions_groups_detailed_statistics.spec.ts

```diff
@@ -117,8 +125,18 @@ export default function ApiTest({ getService }: FtrProviderContext) {
         let metricsStatistics: TransactionsGroupsDetailedStatistics;
         before(async () => {
           [metricsStatistics, transactionsStatistics] = await Promise.all([
-            callApi({ query: { kuery: 'processor.event : "metric"' } }),
-            callApi({ query: { kuery: 'processor.event : "transaction"' } }),
+            callApi({
+              query: {
+                documentType: ApmDocumentType.TransactionMetric,
+                rollupInterval: RollupInterval.OneMinute,
+              },
+            }),
```

Comment:
What about also testing the other rollup intervals (10m, 60m)?

Reply:
Thank you for highlighting this, i have added tests to check data available for 10m and 60m as well